nextflow_schema.json

```diff
@@ -387,6 +391,5 @@
         {
             "$ref": "#/$defs/generic_options"
         }
-    ],
-    "properties": {}
```

Comment:
Why did you remove this?

Reply:
because it was an empty Chunk of code that is written by nf-core schema build. it can be there or not, it will not change anything